.github/workflows/gh_pages.yaml

```diff
@@ -31,12 +31,13 @@ jobs:
           path: jsdoc
 
       - name: Generate JS docs
+        continue-on-error: true
         run: yarn docs
 
       - name: Publish JS docs
         run: |
           TAG=`git describe --tags`
-          cd jsdoc
+          cd jsdoc/
```

Comment:
Is this change needed?

Reply:
Not really, just to conform to the other files